SU2_CFD/src/solvers/CHeatSolver.cpp

```diff
@@ -748,12 +747,14 @@ void CHeatSolver::SetTime_Step(CGeometry *geometry, CSolver **solver_container,
       const su2double Area2 = GeometryToolbox::SquaredNorm(nDim, Normal);
 
       if (flow) {
-        const su2double laminar_viscosity = 0.5 * (flow_nodes->GetLaminarViscosity(iPoint) +
-                                                   flow_nodes->GetLaminarViscosity(jPoint));
+        const su2double thermal_conductivity = 0.5 * (flow_nodes->GetThermalConductivity(iPoint) +
+                                                   flow_nodes->GetThermalConductivity(jPoint));
+        const su2double Cp = 0.5 * (flow_nodes->GetSpecificHeatCp(iPoint) +
+                                                flow_nodes->GetSpecificHeatCp(jPoint));
```

Comment:
formating

Reply:
done, please let me know if this formatting is fine. Thanks!!